ksqldb-functional-tests/src/test/resources/query-validation-tests/partition-by.json

```diff
@@ -1,5 +1,121 @@
 {
   "tests": [
+    {
+      "name": "only key column - select star",
```

Comment:
can we add a test here to partition by an expression?

Reply:
Good catch!